dashboard/src/components/LoginForm/LoginForm.tsx

```diff
@@ -97,6 +71,64 @@ class LoginForm extends React.Component<ILoginFormProps, ILoginFormState> {
   private handleTokenChange = (e: React.FormEvent<HTMLInputElement>) => {
     this.setState({ token: e.currentTarget.value });
   };
+
+  private oauthLogin = () => {
+    return (
+      <>
+        <p>
+          Your cluster operator has enabled login via an authentication provider.{" "}
+          <a
+            href="https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider.md"
+            target="_blank"
+          >
+            Click here
+          </a>{" "}
+          for more info about using authentication providers with Kubeapps.
+        </p>
+        <a href={this.props.loginURI} className="button button-accent">
```

Comment:
how is this a link and not a button? (it's okay if it's the same, I am just curious)

Reply:
Do you mean why, rather than how? It's a plain link both in function (ie. just linking to a URI not handled by the app, without the `/#/...` hash uri). But yes, the class ensures it's the same in appearance.

Reply:
yes, well, I was just surprised that a link with the button classes looks exactly the same than the button